Modix.Data/Repositories/InfractionRepository.cs

```diff
@@ -267,6 +277,33 @@ public async Task<bool> TryRescindAsync(long infractionId, ulong rescindedById)
             return true;
         }
 
+        /// <inheritdoc />
+        public async Task<bool> TryRestoreAsync(long infractionId, ulong restoredById)
+        {
+            var entity = await ModixContext.Infractions
+                .Where(x => x.Id == infractionId)
+                .FirstOrDefaultAsync();
+
+            if ((entity == null) || (entity.RescindActionId != null))
```

Comment:
`if ((entity == null) || (entity.RescindActionId != null))` appears incorrect. Consider replacing it with `if ((entity == null) || (entity.RestoreActionId != null))`

Reply:
Fixed

Reply:
Did you push that, @QuesosNachos I see `RescindAction` still? Unless this end hasn't updated.